x-pack/plugins/observability_solution/observability_ai_assistant_app/public/functions/visualize_esql.tsx

```diff
@@ -259,150 +259,156 @@ export function VisualizeESQL({
 
   return (
     <>
-      {!isLensInputTable && (
-        <EuiFlexGroup direction="column">
-          {Boolean(errorMessages?.length) && (
-            <>
-              <EuiText size="s">
-                {i18n.translate('xpack.observabilityAiAssistant.lensESQLFunction.errorMessage', {
-                  defaultMessage: 'There were some errors in the generated query',
-                })}
-              </EuiText>
-              <EuiDescriptionList data-test-subj="observabilityAiAssistantErrorsList">
-                {errorMessages?.map((error, index) => {
-                  return (
-                    <EuiDescriptionListDescription key={index}>
-                      <EuiFlexGroup gutterSize="s" alignItems="center">
-                        <EuiFlexItem grow={false}>
-                          <EuiIcon type="error" color="danger" size="s" />
-                        </EuiFlexItem>
-                        <EuiFlexItem grow={false}>{error}</EuiFlexItem>
-                      </EuiFlexGroup>
-                    </EuiDescriptionListDescription>
-                  );
-                })}
-              </EuiDescriptionList>
-            </>
-          )}
-
-          <EuiFlexItem grow={false}>
-            <EuiFlexGroup direction="row" gutterSize="s" justifyContent="flexEnd">
-              <EuiFlexItem grow={false}>
-                <EuiToolTip
-                  content={
-                    isTableVisible
-                      ? i18n.translate(
-                          'xpack.observabilityAiAssistant.lensESQLFunction.visualization',
-                          {
-                            defaultMessage: 'Visualization',
-                          }
-                        )
-                      : i18n.translate('xpack.observabilityAiAssistant.lensESQLFunction.table', {
-                          defaultMessage: 'Table of results',
-                        })
-                  }
-                >
-                  <EuiButtonIcon
-                    size="xs"
-                    iconType={isTableVisible ? 'visBarVerticalStacked' : 'tableDensityExpanded'}
-                    onClick={() => setIsTableVisible(!isTableVisible)}
-                    data-test-subj="observabilityAiAssistantLensESQLDisplayTableButton"
-                    aria-label={
+      <EuiFlexGroup direction="column">
+        {Boolean(errorMessages?.length) && (
```

Comment:
fwiw if you refine with `!!` instead of `Boolean()` you don't have to use `errorMessages?.map`, TS will understand that it's there. but also who cares!

Reply:
Done https://github.com/elastic/kibana/pull/184923/commits/58f2c3832485dad9f5d25ca044aeef8bfdf5bac8